src/main/resources/assets/computercraft/lua/rom/apis/window

```diff
@@ -270,19 +270,23 @@ function create( parent, nX, nY, nWidth, nHeight, bStartVisible )
     end
 
     local function setTextColor( color )
+	if type(color) == "number" then
+       if type(tHex[color]) == "string" then
         nTextColor = color
         if bVisible then
             updateCursorColor()
         end
+       else
+        error("Colour out of range", 2)
+       end
+    else
+       error("Color must be a number",2)
```

Comment:
Minor issue, but this line spells it "color" and the above is "colour", being consistent with whatever old errors do would be better.